src/components/Header/header.css

```diff
@@ -0,0 +1,5 @@
+.header {
+    background-image: url('../../../public/images/cover.c5eecd4b.png');}
+
+.search-box {
+    max-width: 740px;}
```

Comment:
```suggestion
.header {
    background-image: url('../../../public/images/cover.c5eecd4b.png');
}

.search-box {
    max-width: 740px;
}
```
Ngoài ra chỗ này đang có cảnh báo endline của github nè e
![image](https://github.com/user-attachments/assets/034dc469-1567-4529-947c-e0cfe84e2c85)